renovate.json

```diff
@@ -5,7 +5,8 @@
   "ignorePaths": [
     "example-problems/**",
     "enforcer-rules/src/it/**",
-    "boms/convergence-check"
+    "boms/convergence-check",
+    "verboseTree-maven-plugin/src/it/**"
```

Comment:
For integration test, this exclusion keeps the versions fixed so that the behavior of the tool is kept the same.